code/game/objects/items/dyekit.dm

```diff
@@ -1,6 +1,7 @@
 /obj/item/dyespray
 	name = "hair dye spray"
 	desc = "A spray to dye your hair any gradients you'd like."
+	w_class = WEIGHT_CLASS_TINY
```

Comment:
it is 10:30 pm and i cannot be fucked to ask what this is so i'm just gonna merge it